spec/classes/vault_spec.rb

```diff
@@ -35,780 +35,788 @@
         it { is_expected.to contain_class('vault::config') }
         it { is_expected.to contain_class('vault::service').that_subscribes_to('Class[vault::config]') }
 
-        it { is_expected.to contain_service('vault')
-          .with_ensure('running')
-          .with_enable(true)
+        it {
+          is_expected.to contain_service('vault').
+          with_ensure('running').
+          with_enable(true)
         }
         it { is_expected.to contain_user('vault') }
         it { is_expected.to contain_group('vault') }
         it { is_expected.not_to contain_file('/data/vault') }
 
-        context "do not manage user and group" do
-          let(:params) {{
-            :manage_user => false,
-            :manage_group => false
+        context 'do not manage user and group' do
+          let(:params) {
+            {
```

Comment:
Is this formatting style suggested by rubocop? Just looks a little odd.

Reply:
sometimes rubocop confuses itself. multiline blocks should be do...end, not {}. I will update it

Reply:
updated